src/components/library/library.jsx

```diff
@@ -60,8 +60,15 @@ const getItemImageSource = function (item) {
         };
     }
 
-    // TODO: adjust libraries to be more storage-friendly; don't use split() here.
-    const md5ext = item.md5 || item.baseLayerMD5;
+    if (item.assetId && item.dataFormat) {
+        return {
+            assetId: item.assetId,
+            assetType: getAssetTypeForFileExtension(item.dataFormat)
+        };
+    }
+
+    // legacy
+    const md5ext = item.md5ext || item.md5 || item.baseLayerMD5;
```

Comment:
are these all still possible?

Reply:
We don't seem to generally use `baseLayerMD5` anymore, but seems safest to keep it here for now. We can revisit when we do a library data cleanup.